modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+You can upgrade the platform version of a managed {sno} cluster with the {lcao}.
+Using a generated seed image allows you to skip multiple minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a seed cluster, which is deployed and configured with the desired {product-title} and applications. You can upgrade the platform version on any {sno} that matches the cluster profile of the seed image.
```

Comment:
```suggestion
You generate the seed image from a seed single-node cluster, which is deployed and configured with the desired {product-title} and applications. You can upgrade the platform version on any {sno} that matches the cluster profile of the seed image.
```

The word cluster inherently implies multiple nodes, so mentioning it without "single-node" could give the impression that "Seed cluster" refers to the hub, or something else, that is not single-node.